packit.spec

```diff
@@ -13,6 +13,7 @@ BuildArch:      noarch
 BuildRequires:  python3-devel
 BuildRequires:  python3-click-man
 BuildRequires:  python3-GitPython
+BuildRequires:  python3-gnupg
```

Comment:
put it to the recipe as well, please

Reply:
Oh, it takes me a while to find the `./files/install-rpm-packages.yaml`.  Yes, I'll add that there.